runelite-client/src/main/java/net/runelite/client/plugins/minimap/MinimapPlugin.java

```diff
@@ -187,4 +259,80 @@ private void restoreOriginalDots()
 
 		System.arraycopy(originalDotSprites, 0, mapDots, 0, mapDots.length);
 	}
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() == WidgetID.MINIMAP_GROUP_ID)
+		{
+			if (config.leftAlignOrbs() != AlignmentMode.OFF)
+			{
+				if (config.leftAlignOrbs() == AlignmentMode.BOTH)
+				{
+					moveOrbs();
+				}
+				else if (!client.isResized() && config.leftAlignOrbs() == AlignmentMode.FIXED)
+				{
+					moveOrbs();
+				}
+				else if (client.isResized() && config.leftAlignOrbs() == AlignmentMode.RESIZEABLE)
+				{
+					moveOrbs();
+				}
+			}
+		}
+	}
+
+	private void moveOrbs()
```

Comment:
This method and `resetOrbs()` are very similar. Their code should probably be extracted to another private method which takes some parameter to determine which of the differing code should be run.